plugins/linuxv2/model/interfaces/interfaces.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package interfaces;
+
+import "github.com/ligato/vpp-agent/plugins/linuxv2/model/namespace/namespace.proto";
+
+enum LinuxInterfaceType {               /* Available interface types */
```

Comment:
It would be much cleaner to have this defined inside `LinuxInterface` message. The generated type will also be easier to read. `interfaces.LinuxInterface_Type` vs. `interfaces.LinuxInterfaceType`.